README.md

```diff
@@ -96,7 +96,6 @@ source scripts/configure_trillian.sh && createLog && createMap
 ```
 
 3. Run Key Transparency
-- `docker-compose build kt-signer`
```

Comment:
This looks like we might need to update the deploy script. I'll take care of that in case this is necessary.